build/dependencies.props

```diff
@@ -1,6 +1,11 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
+    <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
     <CoreFxVersion>4.3.0</CoreFxVersion>
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <TestSdkVersion>15.0.0</TestSdkVersion>
+    <XunitVersion>2.2.0</XunitVersion>
+    <DotNetCliUtilsVersion>1.0.0-preview2-003121</DotNetCliUtilsVersion>
```

Comment:
Might not need this either, but requires https://github.com/aspnet/MvcPrecompilation/blob/dev/test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/CommandResultExtensions.cs to be removed.